### DISCUSSION TITLE
tgcf 0.1.25

### DISCUSSION THREAD

**Discussion by aahnik** [aahnik/tgcf#70] [General]
- [x] fix issue #62 
- [x] merge pr #65  (make cloud deployment possible )
- [x] how to [deploy to heroku](https://github.com/aahnik/tgcf/wiki/Deploy-to-Heroku)
- [x] how to [run on android](https://github.com/aahnik/tgcf/wiki/Run-on-Android-using-Termux)
- [x] how to [run on windows](https://github.com/aahnik/tgcf/wiki/Run-tgcf-on-Windows)

<hr /><em>This discussion was created from the release <a href='https://github.com/aahnik/tgcf/releases/tag/v0.1.25'>tgcf 0.1.25</a>.</em>

**Comment by RebbePod** [comment]
Awesome! Looking forward to test!
Thank you so much @aahnik
